torch/csrc/jit/codegen/cuda/ir_nodes.cpp

```diff
@@ -869,11 +865,27 @@ std::vector<int64_t> TensorDomain::mapDomainCtoP(
       continue;
     }
 
-    consumer_to_producer[itc] = itp;
+    // consumer_to_producer[itc] = itp;
```

Comment:
could you just remove this comment please?

Reply:
Thanks. Done.